### PR TITLE
Create calculator form

### PR DESCRIPTION
**Pull request by mrocha51248**

### CODE REVIEW

File: src/Controller/CalculatorController.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    #[Route('/save', name: 'save')]
+    public function save(Request $request): Response
+    {
+        $order = new Order();
+        $form = $this->createForm(CalculatorType::class, $order);
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            $order->setSavedAt(new DateTime());
+            $entityManager = $this->getDoctrine()->getManager();
```
Comment (rocambille) [inline src/Controller/CalculatorController.php]: dependency injection is a better practice ;)

File: src/Controller/CalculatorController.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route('/', name: 'calculator_')]
+class CalculatorController extends AbstractController
+{
+    #[Route('/', name: 'index')]
+    public function index(Request $request): Response
```
Comment (rocambille) [inline src/Controller/CalculatorController.php]: what is the difference between these 2 methods?

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/Controller/CalculatorController.php (hunk 1 above)
Comment (mrocha51248) [reply src/Controller/CalculatorController.php]: fixed

File: src/Controller/CalculatorController.php (hunk 2 above)
Comment (mrocha51248) [reply src/Controller/CalculatorController.php]: as we discussed, i've modified the controller to let the client handle calculations before submitting

**Pull request merged**